extensions/activemq/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -0,0 +1,28 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+---
+name: "Camel Quarkus ActiveMQ"
+description: "Camel Quarkus ActiveMQ Support"
+metadata:
+  keywords:
+    - "camel"
+    - "ActiveMQ"
```

Comment:
I think used lower case quite consequently so far.

Reply:
I'll change it in a follow up commit. AFAIK, the keyword search on code.quarkus.io is case insensitive, so it'll still work fine with a mixture of upper & lower case.